pkgs/by-name/ca/cargo-plumbing/package.nix

```
@@ -0,0 +1,38 @@
+{
+  fetchFromGitHub,
+  lib,
+  openssl,
+  pkg-config,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "cargo-plumbing";
+  version = "0.0.3";
+
+  src = fetchFromGitHub {
+    owner = "crate-ci";
+    repo = "cargo-plumbing";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-x8xH7XH91FtOn5knVL7mkcDTGvXtVVL70HIi8V9z54o=";
+  };
+
+  cargoHash = "sha256-16rY8uk9ViEaYIqiZHHU1UApAdNXAETqgFzUWNto6po=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ];
+
```

```suggestion

  passthru.updateScript = nix-update-script { };

```

Add `nix-update-script`